src/main/java/teammates/common/util/HibernateUtil.java

```diff
@@ -110,6 +110,7 @@ public static void buildSessionFactory(String dbUrl, String username, String pas
 
         Configuration config = new Configuration()
                 .setProperty("hibernate.dialect", "org.hibernate.dialect.PostgreSQLDialect")
+                .setProperty("hibernate.jdbc.batch_size", "50")
```

Comment:
I'm not sure if we want to commit this to master, as it's only for migration purposes. Maybe we could comment this line for master and put another comment on the migration script to remember to uncomment this line.

Reply:
Is there a way to change the config in the script? Otherwise I think we can put a comment in script and uncomment this line on master.

Reply:
I don't think so. I've added the notes